ddl/ddl_api.go

```diff
@@ -1359,13 +1384,23 @@ func handleTableOptions(options []*ast.TableOption, tbInfo *model.TableInfo) err
 	return nil
 }
 
-func hasAutoIncrementColumn(tbInfo *model.TableInfo) bool {
+func hasAutoIncrementColumn(tbInfo *model.TableInfo) (bool, string) {
 	for _, col := range tbInfo.Columns {
 		if mysql.HasAutoIncrementFlag(col.Flag) {
-			return true
+			return true, col.Name.L
 		}
 	}
-	return false
+	return false, ""
+}
+
+func collectAutoIncrementColumns(tbInfo *model.TableInfo) map[string]struct{} {
```

Comment:
This function is not used?

Reply:
yes, this function is useless after some modifications. I'll remove this. thx